apps/spruce/src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -51,7 +52,7 @@ export const AuxiliaryDropdown: React.FC<AuxiliaryDropdownProps> = ({
     },
     {
       "data-cy": "auxiliary-dropdown-project-health",
-      text: "Project Health",
+      text: <>Project Health <Badge variant="yellow">Deprecated</Badge></>,
```

Comment:
I mean the component itself needs to render side by side use flex box to fix this